specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,102 @@
+|                           | Data Processing                   | Integration and transformation tasks to prepare data for analysis.                                                                                                                                     |
+|                           | Search                            | Discovery of information by indexing and retrieving data from various sources.                                                                                                                         |
+|                           | Streaming Analytics               | Real-time data stream processes to detect patterns, trends, and anomalies as they occur.                                                                                                               |
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                             |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                    |
+| Compute                   | Containers                        | Management and orchestration of containerized compute platforms.                                                                                                                                       |
+|                           | End User Computing                | Virtualized desktop infrastructure and device / endpoint management.                                                                                                                                   |
+|                           | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                                                                                                            |
+|                           | Serverless Compute                | Enablement of compute capabilities without provisioning or managing servers.                                                                                                                           |
+|                           | Virtual Machines                  | Computing environments created by abstracting resources from physical hardware.                                                                                                                        |
+|                           | Other (Compute)                   | Compute services that do not fall into one of the defined subcategories.                                                                                                                               |
```

Comment:
If anything doesn't fit existing categories, they fit well into the 'Other ($Category)' subcategory 😄 

In the case of Spring Apps, I don't think that's a compute service. They seem similar to Elastic Beanstalk type services in AWS where you combine a bunch of services (compute, db, networking etc.) to host applications. This may be something we need to introduce as a top level category or find another place. For now, I'm ok with not supporting this explicitly and adding support based on need.

Reply:
We discussed a place for items such as AWS Elastic Beanstalk, AWS Lightsail, and Azure Spring Apps.  I did not hear a consensus (some said net-new category, some said `Compute`, some said `Web`, some said `Business Applications`), and so I did not pursue this further, as the providers could choose any of the existing categories and not subcategorize.

Reply:
Discussed this further, and we need to revisit as a group in the next TF1 meeting.  Current proposal is either to expand the scope of `Compute > Serverless Compute` to be inclusive of such services or 2) add a new subcategory.  But Spring Apps will have to stay in Compute from Microsoft's perspective, so the subcategory ideally lives there.

Reply:
Discussed in today's TF1 meeting; landed on adding to `Web > Web Platforms`.  Resolving.